src/main/java/core/basesyntax/service/FruitFileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitFileReaderImpl implements FruitFileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> result = new ArrayList<>();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(filePath));
+            bufferedReader.readLine();
```

Comment:
let's make this service reusable - we are now skipping the first line here
we should better do this in another place - so that we can reuse our reader in different situations